schemas/BaseElements.xsd

```diff
@@ -1272,6 +1272,19 @@
 												<xs:element minOccurs="0" name="UsedForWholeBuildingVentilation" type="HPXMLBoolean"/>
 												<xs:element minOccurs="0" name="UsedForSeasonalCoolingLoadReduction" type="HPXMLBoolean"/>
 												<xs:element minOccurs="0" name="UsedForGarageVentilation" type="HPXMLBoolean"/>
+												<xs:element minOccurs="0" name="IsSharedMultifamilySystem" type="HPXMLBoolean"/>
+												<xs:element minOccurs="0" name="SharedSystemRatedFlowRate" type="HPXMLDouble">
+													<xs:annotation>
+														<xs:documentation>[CFM] as rated by manufacturer</xs:documentation>
+													</xs:annotation>
+												</xs:element>
+												<xs:element minOccurs="0" name="SharedSystemFanPower" type="HPXMLDouble">
+													<xs:annotation>
+														<xs:documentation>[W]</xs:documentation>
+													</xs:annotation>
+												</xs:element>
+												<xs:element minOccurs="0" name="FractionOutdoorAir" type="HPXMLDouble"> </xs:element>
+												<xs:element minOccurs="0" name="FractionRecirculation" type="HPXMLDouble"> </xs:element>
```

Comment:
Do we really need both oa and recirc fractions?

Reply:
Not necessarily needed but probably reasonable to have both in case one tool collects one input and another tool collects the other.

Reply:
Should probably also add notes describing these two inputs and declaring that they should sum to 1.